.github/workflows/ci.yml

```diff
@@ -1,15 +1,16 @@
 name: CI
 
 on:
+  pull_request:
+    types: [synchronize, opened]
```

Comment:
What do you think about the, what seems to me to be more standard:

```
  pull_request:
    branches: [ "v2.0" ]
```

Reply:
under the `pull_request` `branch` is not the target branch, it's the PR's branch, that's why I had to do it that way.

Reply:
I think it is the target branch based on what I'm seeing on https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request ?

Reply:
No, I did actually tested this already, let me change the ci and and show you it doesn't work

Reply:
Dang!! it's working with `branches`, I could swear that `branches` was not working, I had to figure out how to access the target branch name for the condition :( But it seems to run now. Updated @ryanwi